src/components/admin/localization-statuses/LocalizationStatuses.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    EntityLocalization,
+    EntityWithLocalizations,
+    LocalizationLanguage,
+    TranslationStatus,
+} from '../../../types/common/language';
+import styles from './LocalizationStatuses.module.scss';
+
+export interface LocalizationStatusProps<TLocalization extends EntityLocalization> {
```

Comment:
do we need this generics here and in component? double check please
```suggestion
export interface LocalizationStatusProps {
```

Reply:
this generics is necessary